src/test/java/hudson/plugins/active_directory/docker/ActiveDirectoryGenericContainer.java

```diff
@@ -25,7 +27,8 @@ public ActiveDirectoryGenericContainer() {
                 .withFileFromClasspath("named.conf.options", "hudson/plugins/active_directory/docker/TheFlintstonesTest/TheFlintstones/named.conf.options")
                 .withFileFromClasspath("sssd.conf", "hudson/plugins/active_directory/docker/TheFlintstonesTest/TheFlintstones/sssd.conf")
                 .withFileFromClasspath("supervisord.conf", "hudson/plugins/active_directory/docker/TheFlintstonesTest/TheFlintstones/supervisord.conf"));
-        setWaitStrategy(null);
+        // wait for the custom.sh script to complete successfully
+        setWaitStrategy(new LogMessageWaitStrategy().withRegEx(".*\\Qexited: custom (exit status 0; expected)\\E.*"));
```

Comment:
technically not part of the change, but moves the initialisation wait into the container starting to make failures easier to diagnose (which was needed during the development of this)